src/main/java/org/terasology/math/geom/Winding.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2014 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.math.geom;
+
+/**
+ * @author Martin Steiger
+ */
+public enum Winding {
+
+    CLOCKWISE,
+    COUNTERCLOCKWISE,
+    NONE
```

Comment:
It was used for empty Polygons. But in that case, both windings would be correct. So I removed it.